src/controllers/controllermanager.cpp

```diff
@@ -280,16 +286,12 @@ void ControllerManager::stopPolling() {
 
 void ControllerManager::pollDevices() {
     Trace tracer("ControllerManager::pollDevices");
-    bool eventsProcessed(false);
     // Continue to poll while any device returned data.
-    do {
-        eventsProcessed = false;
-        foreach (Controller* pDevice, m_controllers) {
-            if (pDevice->isOpen() && pDevice->isPolling()) {
-                eventsProcessed = pDevice->poll() || eventsProcessed;
-            }
+    foreach (Controller* pDevice, m_controllers) {
+        if (pDevice->isOpen() && pDevice->isPolling()) {
+            pDevice->poll();
         }
-    } while (eventsProcessed);
```

Comment:
No. 
How is this done on a worst case controller? My RMX 2 limits the data rate on fast spins, in a way that it combines two telegrams: Slow turns +-1 fast turns +-64 on fast turns the maximum I can reach. So no issue here. 

If we assume a worst 1000 tick controller which emit every tick, after this patch, the jog speed is limited to 12 rounds per second, which sounds like a reasonable safety distance to realistic values. 

Are you able to verify it?